lego/apps/users/models.py

```diff
@@ -299,8 +300,17 @@ def email_address(self):
     def profile_picture(self, value):
         self.picture = value
 
+    @PermissionsMixin.is_abakus_member.setter
+    def is_abakus_member(self, add_to_group):
```

Comment:
This is pretty implicit - that changing a boolean on a user causes changes to many to many relations. I guess the alternative would be to have a separate endpoint, or use the existing endpoint for adding memberships to groups - but not sure if it's better.

If this is the best solution it should at least include a few comments describing what it's doing. Maybe the `add_to_group` variable should be called something like `new_value` instead, to signify that it's the value that the user is trying to set `is_abakus_member` to (if I understand it correctly).

Also - where's the tests?

Reply:
Well, it is implicit - but I felt it natural to do the group handling while updating the user. I mean since we are already passing a `is_abakus_member` boolean. I thought of the using the membership endpoint as you mention, but that would require a bit more logic frontend – while this would just be another radiobutton in the user settings page.

I should have added a WIP to this PR – I wanted feedback on this approach before finishing it up with tests.